vuepress/docs/next/tutorials/getting-started/gke-install.md

```diff
@@ -146,31 +146,13 @@ You do not need clustered storage to scale an Entando Application if you schedul
 :::
 
 ### Clustered Storage Using GCP Cloud Filestore
-1. In the left menu of the GCP portal, find the Storage section and select `Filestore` -> `Instances`
+1. In the cluster `DETAILS` tab, under `Features` find the `Filestore CSI driver` property and enable it.
 
-2. Enable the Filestore, if you haven't already
+2. Wait a couple of minutes for the changes to propagate
 
-3. Select `Create Instance`
+3. Ensure also the `Cloud Filestore API` is enabled under `API and services` on the main left menu.
 
-4. Adjust the field values from the defaults as needed. Take note of your instance ID.
-
-5. Once the instance is created on the Filestore main page, note the IP address of your NFS
-
-6. Install the provisioner that creates the StorageClass to enable deployment of Entando Applications. Use the commands below, replacing `YOUR-NFS-IP` and `YOUR-NFS-PATH` with your instance ID and the IP address of your cluster.
-
-  ```
-  helm repo add nfs-subdir-external-provisioner https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner/
-  ```
-  ```
-  helm install nfs-subdir-external-provisioner nfs-subdir-external-provisioner/nfs-subdir-external-provisioner \
-      --set nfs.server=YOUR-NFS-IP \
-      --set nfs.path=YOUR-NFS-PATH
-  ```
-
-  Learn about the provisioner and additional configuration options at
-  <https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner>
-
-7. Verify that your client provisioned successfully. This is indicated by the presence of the storage class `nfs-client` in the output of the following command.
+3. Verify that your client provisioned successfully. This is indicated by the presence of the storage class with the `rwx` suffix in the output of the following command.
```

Comment:
```suggestion
4. Verify that your client was provisioned successfully. This is indicated by the presence of the storage class with `rwx` suffix in the output following the command.
```

Reply:
unsure of what was meant by 'the following command'... is the presence of the storage class with the xxx suffix supposed to follow the command? that's what my update suggests

Reply:
by `the following command` I mean `kubectl get sc` which returns the list of the new generated storage classes